src/app/components/bag/TotalAmount.tsx

```diff
@@ -0,0 +1,35 @@
+"use client";
+import { useAppSelector } from "@/redux/store";
+
+import React, { useState } from "react";
+
+function TotalAmount() {
+
+  let amount = 0;
+  amount = useAppSelector((state) => state.reducers.totalAmount);
```

Comment:
any reason not to write it like
let amount = 0;
  amount = useAppSelector((state) => state.reducers.totalAmount);
  
  let amount = useAppSelector((state) => state.reducers.totalAmount);